pages/1.13/administering-clusters/package-registry/index.md

```diff
@@ -111,16 +112,16 @@ The final `s3` config should something like this:
 
 <p class="message--note"><strong>NOTE: </strong>You must override the value for the properties <code>bucket</code>, <code>path</code> and <code>endpoint</code> to match the S3 configuration.</p>
 
-## Serivice namespacing and secrets
+## Service namespacing and secrets
 
-By default, package registry is installed as a marathon app with `dcos-registry` as its id. This name has somewhat unique significance because the `dockerd` on agents are configured to trust the package registry instance at `dcos-registry.marathon.l4lb.thisdcos.directory:443`. If you decide to change this name, you need to configure the `dockerd` to trust the custom name your registry is deployed at `<your-custom-name>.marathon.l4lb.thisdcos.directory:443`. For e.g. if you install registry under `/my/custom/dcos-registry` namespace, then ensure registry is accessible at `https://mycustomdcos-registry.marathon.l4lb.thisdcos.directory` (internal to cluster):
+By default, package registry is installed as a marathon app with `dcos-registry` as its ID. This name has unique significance because the `dockerd` on agents are configured to trust the package registry instance at `dcos-registry.marathon.l4lb.thisdcos.directory:443`. If you decide to change this name, you need to configure the `dockerd` to trust the custom name your registry is deployed at `<your-custom-name>.marathon.l4lb.thisdcos.directory:443`. For example, if you install package registry under `/my/custom/dcos-registry` namespace, then ensure that the registry is accessible at `https://mycustomdcos-registry.marathon.l4lb.thisdcos.directory` (internal to cluster):
 
 ```bash
 curl -k https://mycustomdcos-registry.marathon.l4lb.thisdcos.directory
 {"checks":{"/repo":{"Healthy":{"message":"Able to find 0 package(s)."}}}}
 ```
 
-You would also require your `dockerd` to trust the above domain name. To configure `dockerd` to trust package registry at `mycustomdcos-registry.marathon.l4lb.thisdcos.directory`, you can execute:
+You must also require your `dockerd` to trust the above domain name. To configure `dockerd` to trust package registry at `mycustomdcos-registry.marathon.l4lb.thisdcos.directory`, execute:
```

Comment:
Write as `You must also require your `dockerd` to trust the above domain name. To configure `dockerd` to trust package registry at `mycustomdcos-registry.marathon.l4lb.thisdcos.directory`, execute the following command:`

Reply:
I need clarification on this. Let's come back to it.

Reply:
Done